internal/installer/elasticagent_exe.go

```diff
@@ -0,0 +1,162 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I see there is a ZIP installer, but cannot find the EXE one: https://artifacts-api.elastic.co/v1/search/8.0.0-SNAPSHOT/elastic-agent?x-elastic-no-kpi=true

```json
"elastic-agent-8.0.0-SNAPSHOT-windows-x86_64.zip":{"url":"https://snapshots.elastic.co/8.0.0-943ef2c0/downloads/beats/elastic-agent/elastic-agent-8.0.0-SNAPSHOT-windows-x86_64.zip","sha_url":"https://snapshots.elastic.co/8.0.0-943ef2c0/downloads/beats/elastic-agent/elastic-agent-8.0.0-SNAPSHOT-windows-x86_64.zip.sha512","asc_url":"https://snapshots.elastic.co/8.0.0-943ef2c0/downloads/beats/elastic-agent/elastic-agent-8.0.0-SNAPSHOT-windows-x86_64.zip.asc","type":"zip","architecture":"x86_64","os":["windows"]}
```

Reply:
Ah right, I was going based off elastic-agent.exe but we can change this to zip installer